src/osd/OSD.cc

```diff
@@ -4363,18 +4363,20 @@ void OSD::final_init()
     "");
   ceph_assert(r == 0);
   r = admin_socket->register_command(
-    "pg "			   \
-    "name=pgid,type=CephPgid "	   \
-    "name=cmd,type=CephChoices,strings=scrub " \
-    "name=time,type=CephInt,req=false",
+    "pg "
+    "name=pgid,type=CephPgid "
+    "name=cmd,type=CephChoices,strings=scrub "
+    "name=time,type=CephInt,req=false "
+    "name=force,type=CephBool,req=false",
```

Comment:
ACK. So we're just adding the `force` param. Other lines are changes for the sake of backslash cleaning.

Reply:
indeed so.